mm2src/coins/utxo/utxo_common.rs

```diff
@@ -5190,42 +5258,91 @@ fn test_tx_v_size() {
 }
 
 #[test]
-fn test_generate_taker_fee_tx_outputs() {
-    let amount = BigDecimal::from(6150);
-    let fee_amount = sat_from_big_decimal(&amount, 8).unwrap();
+fn test_generate_taker_fee_tx_outputs_with_non_kmd_burn() {
+    let client = UtxoRpcClientEnum::Native(NativeClient(Arc::new(NativeClientImpl::default())));
+    let mut fields = utxo_coin_fields_for_test(client, None, false);
+    fields.conf.ticker = "MYCOIN1".to_owned();
+    let coin = utxo_coin_from_fields(fields);
+
+    let fee_amount = BigDecimal::from(6150);
+    let burn_amount = &(&fee_amount / &BigDecimal::from_str("0.75").unwrap()) - &fee_amount;
+    let fee_uamount = sat_from_big_decimal(&fee_amount, 8).unwrap();
+    let burn_uamount = sat_from_big_decimal(&burn_amount, 8).unwrap();
 
     let outputs = generate_taker_fee_tx_outputs(
-        8,
-        &AddressHashEnum::default_address_hash(),
-        &DexFee::Standard(amount.into()),
+        &coin,
+        &DexFee::create_from_fields(fee_amount.into(), burn_amount.into(), "MYCOIN1"), // TODO: Add test for ETH and DexFee::Standard
```

Comment:
Do you plan to add this test in this PR? Just a reminder about this todo otherwise we can forget about it.

Reply:
Added a test for utxo standard dex fee f35be0047ba24688c8484e8f264edc3827c8bce6.
Removed comment for EVM (we do not add burning for EVM yet)